src/github.com/ywwg/items/friday_dinner.go

```diff
@@ -0,0 +1,18 @@
+package items
+
+import (
+	plib "github.com/ywwg/packinglib"
+)
+
+var burnDinner = []plib.Item{
+	plib.NewConsumableItem("pulled pork", 8.5, "pounds", []string{"Burn"}, nil),
+	plib.NewConsumableItem("vegan chili", 3.5, "gallons", []string{"Burn"}, nil),
+	plib.NewConsumableItem("pulled jackfruit", 50, "ounces", []string{"Burn"}, nil),
+	plib.NewConsumableItem("shredded cheese", 2, "pounds", []string{"Burn"}, nil),
+	plib.NewConsumableItem("sour cream", 24, "ounces", []string{"Burn"}, nil),
+	plib.NewBasicItem("sarina's cooler", []string{"Camping"}, nil),
```

Comment:
Oh cool so now whenever I go camping I will always bring your cooler :p

j/k, I know I need to figure out things like "one-time" packable items.  There's also the question of how to handle forks of this repo, but let everyone have their own packing lists.